### DISCUSSION TITLE
NextJs - É possível desativar o AMP sem quebrar a UI?

### DISCUSSION THREAD

**Discussion by luiznasciment0** [frontendbr/forum#1930] [Perguntas e Respostas]
Eu preciso desativar o AMP em algumas páginas para coletar métricas/analytics, mas se eu só setar `export const config = { amp: true }` para false, minha UI inteira quebra. Como eu posso fazer isso? É possível?

Muito obrigado :D

**Comment by tiagooliveira08** [comment]
Nunca tentei, acredito que só testando pra saber, tenta fazer no local. 
Mas é necessário mesmo desativar o amp pra coletar essa metricas?
Até pq você pode ter dados inconsistentes já que você está desativando uma funcionalidade importante.

**Comment by luiznasciment0** [comment]
> Nunca tentei, acredito que só testando pra saber, tenta fazer no local.
> Mas é necessário mesmo desativar o amp pra coletar essa metricas?
> Até pq você pode ter dados inconsistentes já que você está desativando uma funcionalidade importante.

Na verdade, por meio de algumas pesquisas, vi que existe uma grande possibilidade de o AMP distorcer as métricas e analytics, por algum motivo. Então a ideia é desativá-lo para tentar ajustar isso, mas sem quebrar a interface haha
